cmd/groups-auto-updater/main.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/test-infra/prow/config/org"
+
+	userV1 "github.com/openshift/client-go/user/clientset/versioned/typed/user/v1"
+
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+// Config holds the information of all group names and their configuration
```

Comment:
This seems like a lot of complexity. Why? We need a tool that takes four(three?) things as input:
 - the peribolos configuration path and the org to copy users from
 - the name of the group to update and a path to a kubeconfig to do it with (or just run it with in-cluster config)

So why not three flags?

Reply:
Yea, let's keep it simple for now. I will remove the config and add the flags.